src/pipx/commands/common.py

```diff
@@ -148,7 +149,8 @@ def get_package_summary(
             f"       Please uninstall and install this package, or reinstall-all to fix."
```

Comment:
In the function definition for get_package_summary you no longer have any use for argument `suffix` and it should be removed.